modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/converter/DateTypeConverter.kt

```diff
@@ -6,8 +6,8 @@ import java.util.Date
 class DateTypeConverter {
 
     @TypeConverter
-    fun toDate(value: Long?): Date? {
-        return if (value == null) null else Date(value)
+    fun toDate(value: Long?): Date {
+        return if (value == null) Date() else Date(value)
```

Comment:
This should stop the following Sentry error.
```
au.com.shiftyjelly.pocketcasts.models.db.dao.EpisodeDao_Impl$44 in call
IllegalStateException
Expected non-null java.util.Date, but it was null.
```
It seems like the database episode has a publish date of null. I can't find how this is possible but this should stop the crash happening when the data is read.

Reply:
Maybe `LogBuffer.log` would be better than just adding the breadcrumb.

Reply:
Excellent idea to log and return a constant date. I have made this change.

Reply:
That log just saved me! It gets called with dates that need to be null. I'm reverting this issue fix.